packages/suite/src/views/dashboard/components/PromoBanner.tsx

```diff
@@ -0,0 +1,209 @@
+import React, { useState } from 'react';
+import styled, { css } from 'styled-components';
+import { SUITE_MOBILE_APP_STORE, SUITE_MOBILE_PLAY_STORE, SUITE_URL } from '@trezor/urls';
+import { Button, Icon, Image, ImageType, Tooltip, variables } from '@trezor/components';
+import { Translation, QrCode, TrezorLink } from '@suite-components';
+import { isWeb } from '@suite-utils/env';
+import { useLayoutSize } from '@suite-hooks/useLayoutSize';
+import { DESKTOP_HORIZONTAL_PADDINGS, MOBILE_HORIZONTAL_PADDINGS } from '@suite-constants/layout';
+
+const Container = styled.div`
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    display: flex;
+    align-items: center;
+    width: 100%;
+    height: 70px;
+    border-top: 1px solid ${({ theme }) => theme.STROKE_GREY};
+    font-size: ${variables.FONT_SIZE.SMALL};
+
+    ${variables.SCREEN_QUERY.MOBILE} {
+        height: 110px;
+        border-radius: 20px;
+        box-shadow: 0px -4px 6px -4px ${({ theme }) => theme.BOX_SHADOW_OPTION_CARD};
+    }
+`;
+
+const promoContainerCss = css`
+    display: flex;
+    align-items: center;
+    flex: 1;
+    gap: 16px;
+    height: 100%;
+    padding: 0 ${DESKTOP_HORIZONTAL_PADDINGS};
+
+    ${variables.SCREEN_QUERY.BELOW_LAPTOP} {
+        padding: 0 ${MOBILE_HORIZONTAL_PADDINGS};
+    }
+`;
+
+const DesktopPromoContainer = styled.div`
+    ${promoContainerCss}
+    border-right: 1px solid ${({ theme }) => theme.STROKE_GREY};
+`;
+
+const MobilePromoContainer = styled.div`
+    ${promoContainerCss}
+
+    ${variables.SCREEN_QUERY.MOBILE} {
+        flex-direction: column;
+        gap: 0;
+        padding: 20px 16px;
+        justify-content: space-between;
+    }
+`;
+
+const OSIcons = styled.div`
+    display: flex;
+    align-self: center;
+    align-items: center;
+    gap: 6px;
+    margin-top: 2px;
+    opacity: 0.7;
+`;
+
+const StyledLink = styled(TrezorLink)`
+    margin-left: auto;
+`;
+
+const DesktopLinkButton = styled(Button)`
+    background: ${({ theme }) => theme.STROKE_GREY};
+    color: ${({ theme }) => theme.TYPE_DARK_GREY};
+    font-size: ${variables.FONT_SIZE.SMALL};
+    transition: opacity 0.15s;
+    opacity: 0.6;
+
+    :hover,
+    :focus {
+        background: ${({ theme }) => theme.STROKE_GREY};
+        opacity: 1;
+    }
+`;
+
+const BadgeContainer = styled.div`
+    display: flex;
+    gap: 10px;
+    margin-left: auto;
```

Comment:
I'd find this sort of alignment a bit clearer as it ties the message to the button:

![Screenshot 2023-05-11 at 14 38 45](https://github.com/trezor/trezor-suite/assets/42465546/29bb1cc5-0849-416c-bacb-9f52da3e45df)

Reply:
was also considering that, @sime u ok with that?

Reply:
58df7ee3c

Reply:
Ideally the copywriters should try to fit the text but I could fix it later if it looks ugly :)